bears/vcs/git/GitCommitBear.py

```diff
@@ -134,6 +145,34 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             body,
             **self.get_issue_checks_metadata().filter_parameters(kwargs))
 
+    def check_auth_email(self, auth_email,
+                         auth_email_check: bool=True,
+                         email_dns_check: bool=True):
+        """
+        Checks validity of author's email
+
+        :param auth_email_check:      If True, it will check validity of
+                                      author's email.
+        :param email_dns_check:       If True, it will confirm the domain
+                                      in author's email is valid.
+        """
+        if not auth_email_check:
+            return
+
+        # Used regex provided in http://emailregex.com/
+        email_check = re.match(r'(^[a-zA-Z0-9_.+-]'
```

Comment:
alright then I will be just using `is_email` to do that. and about using `validate_email`, it was my first try but it was giving Server Error in travis. See [comment](https://github.com/coala/coala-bears/pull/1944#issuecomment-317323142). thats why i shifted to pyisemail